### ISSUE TITLE
Improve FigisMap.console for debugging

### ISSUE THREAD

**Issue by eblondel** [openfigis/FigisMap#40]
FigisMap.console is useful for debugging but because it is called with a custom function, we loose the js code line number where it is invoked within FigisMap. @marcobalestra do you agree that i have a look and possibly improve this?

**Comment by marcobalestra** [comment]
It's a custom function instead of classic Console.log() because Console.log() is not allowed in IE, then its presence would break js on faulty browsers (IE).
There is also the option to check for existence of Console.log method, and define it if missing in order to avoid errors. I didn't do it because the idea was that FigisMap.js code was populating all and only the FigisMap object.

**Comment by eblondel** [comment]
i'm not questioning the function itself, i knew about it was intended to
give logs even when there is no window.console, just wondering if there
would be some way to handle js line involved and not the same js line
pointing to FigisMap.console.. but maybe there is no simple way to do that..

**Comment by marcobalestra** [comment]
Agreed... A few interesting considerations: http://stackoverflow.com/questions/2343343/how-can-i-determine-the-current-line-number-in-javascript
